n_agents: 1
build_acados: true
drone_properties:
  mass: 0.0282
  Ixx: 1.6572e-05
  Iyy: 1.6656e-05
  Izz: 2.9262e-05
  cm: 2.4e-06
  arm_length: 0.044
  motorConstant: 1.7965e-08
  momentConstant: 9.1785e-07
mpc:
  horizon: 1.0
  num_steps: 50
  control_update_rate: 50
  plot_trajectory: true
  model_type: full_model
simple_model:
  attitude_time_constant: 0.5
full_model:
  kappa: 8
  use_predictor: false
delay_relay:
  enabled: true
  delay_ms: 25
  jitter_ms: 5
path_planner:
  trajectory_type: step
